package.json

```diff
@@ -127,7 +128,7 @@
     "timecop": "0.36.0",
     "tree-view": "0.217.0-7",
     "update-package-dependencies": "0.11.0",
-    "welcome": "0.36.2",
+    "welcome": "0.36.3",
```

Comment:
This just needed changes to the tests (change `atom.workspace.getPanes()` to `atom.workspace.getCenter().getPanes()`).